meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor

```diff
@@ -149,8 +149,5 @@ echo "SUPERVISOR_IMAGE=$image_name:$tag" > $UPDATECONF
 echo "Start supervisor..."
 systemctl start resin-supervisor
 
-# Mark supervisor as working. This version will run when the device reboots.
-echo "Mark pulled tag as latest..."
-$DOCKER tag -f "$image_name:$tag" "$image_name:latest"
 sed --in-place "s|SUPERVISOR_IMAGE=.*|SUPERVISOR_IMAGE=$image_name|" /etc/resin-supervisor/supervisor.conf
```

Comment:
@willnewton @agherzan sed does it by default, but here we have two separate sed calls so it will not be atomic...